### PR TITLE
New Syndicate redesigns

### PR DESCRIPTION
**Pull request by beccabunny**
## Summary
This PR is a new attempt at the syndicate redesigns. Examples shown next:
comparison marrow
*Old and new marauder arrow.*
comparison star barge
*Old and new star barge.*
comparison vanguard
*Old and new vanguard.*
comparison wasp
*Old and new wasp.*

## PR Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I uploaded the necessary image, blend, and texture assets here: https://drive.google.com/file/d/1qXCS_1zsU9UL_l9ufQZucZLgdTJk2Bro/view?usp=sharing
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/157

### CODE REVIEW

**Comment by Zitchas** [comment]
Could we get a top-down comparision set too please?

**Comment by MCOfficer** [comment]
Sourced from the hdpi PR - note that these may be subject to change.

<details>
<summary> Click to open </summary>

Old ships on the left, new ones on the right. Note that sometimes the new ones are a bit blurry, because they are scaled to the old image's height - for an **accurate, uptodate** comparison please refer to the rich diff.

arrow
image


barb
image


barbp
image


bulk freighter
image


freighter
image


marrowe
image


marrows
image


marroww
image


pointedstick vanguard
image


protector
image


star barge
image


vanguard
image


wasp
image


</details>

**Comment by beccabunny** [comment]
Aaaand MCO ninjaed me of three minutes. Rip.

**Comment by ravenshining** [comment]
The Bulk Freighter looks nice up close, but in-game it's one big boring rectangle, the old one had a much more interesting silhouette.  Also, it looks like more of a container ship than a bulk freighter now.  A nice new ship to have perhaps, but not a direct replacement.

I don't like the Barbs either, but they are tiny, rare, and not as bad, so they don't matter as much.

**Comment by Zitchas** [comment]
In regards to the bulk freighter, I agree with that. I prefer the old cargo pods rather than the new multi-modal-containers.

https://www.yusen-logistics.com/sites/default/files/global/services/intermodal_and_multimodal.jpg

vs

https://upload.wikimedia.org/wikipedia/commons/7/7c/Unloading_JAL_747.jpg

Personally, the aircraft pods looks *much* more interesting than large metal cubes. I hadn't even thought of Ravenshining's observation that this turns the bulk freighter into a giant rectangle.

That being said, the nose of the bulk freighter, the spine, and engines are a really good improvement, and I like those parts a lot.

**Comment by ravenshining** [comment]
Oh, yeah, the spine and the engine block look great.  Not a fan of the cockpit but it's not a dealbreaker.

When I look at the old bulk freighter, I think of ships like this:

image

And I will happily work on updating the appearance of the old side pods in that manner.  If I were as good as Becca at modelling, I might add some built-in unloading equipment like this ship has:

image

I could get on that - and some tweaks to improve the silhouette of this container variant - faster if someone could post the bulk freighter assets separately, rather than one as part of one 57 MB zip file. My jungle internet is a bit janky and expensive when it comes to large downloads.

File: copyright [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+﻿mat: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```
Comment (Amazinite) [inline copyright:1]: Damn.

**Review by Amazinite — CHANGES_REQUESTED** [review]
Stream comments:
* Perhaps touch up the Bulk Freighter cargo pods to make them look less flat/"like a ramp" (- MCO). Bring back the forked cargo pod?

File: copyright (hunk 1 above)
Comment (beccabunny) [reply copyright]: resolved

**Comment by beccabunny** [comment]
@Amazinite changed the bulk freighter with the last one I had posted, it should be all good now

**Comment by ravenshining** [comment]
That is looking good - it doesn't take all the suggestions but enough of them to not look boring - though I still say it should be a new ship or variant, not a replacement.

File: copyright [hunk 2]
```diff
@@ -607,6 +607,8 @@ Comment: Derived from works by Michael Zahniser (under the same license) and det
 
 Files:
  images/ship/boxwing*
+ images/ship/barb*
+ images/thumbnail/boxwing*
  images/thumbnail/boxwing*
```
Comment (Amazinite) [inline copyright:612]:
```suggestion
 images/ship/barb*
 images/ship/boxwing*
 images/thumbnail/barb*
 images/thumbnail/boxwing*
```

File: copyright [hunk 3]
```diff
@@ -628,6 +630,13 @@ Files:
  images/ship/leviathan*
  images/ship/shuttle*
  images/ship/star?queen*
+ images/ship/localworldship*
+ images/ship/arrow*
+ images/ship/bulk?freighter*
+ images/ship/freighter*
+ images/ship/protector*
+ images/ship/star?barge*
+ images/ship/wasp*
```
Comment (Amazinite) [inline copyright:639]: Missing copyright entry for the thumbnails of these ships.

File: copyright [hunk 4]
```diff
@@ -643,20 +652,33 @@ Files:
...
 Comment: Derived from works by Michael Zahniser (under the same license) and detailed by Becca Tommaso (tommasobecca03@gmail.com).
 
+Files:
+ images/ship/pointedstick vanguard*
+Copyright: Maximilian Korber
+License: CC-BY-SA-4.0
+Comment: Derived from works by Nate Graham (under the same license) and detailed by Becca Tommaso (tommasobecca03@gmail.com).
+
+Files:
+ images/ship/vanguard*
```
Comment (Amazinite) [inline copyright:677]: Missing thumbnail entry.

**Review by Amazinite — CHANGES_REQUESTED** [review]

File: copyright (hunk 3 above)
Comment (beccabunny) [reply copyright:639]: done

**Pull request merged**